docs/configuration/index.md

```diff
@@ -4,61 +4,70 @@ The following table lists the configurable parameters of the Codacy chart and th
+| `global.metricsdb.postgresqlPassword`        | Hostname of the Postgresql server        | None            |
+| `global.metricsdb.host`                      | Hostname of the Postgresql server        | None            |
+| `global.metricsdb.service.port`              | Port of the Postgresql server            | `5432`          |
+| `global.filestoredb.postgresqlUsername`      | Username of the Postgresql server        | `codacy`        |
+| `global.filestoredb.postgresqlDatabase`      | Database name of the Postgresql server   | `filestore`     |
+| `global.filestoredb.postgresqlPassword`      | Hostname of the Postgresql server        | None            |
+| `global.filestoredb.host`                    | Hostname of the Postgresql server        | None            |
+| `global.filestoredb.service.port`            | Port of the Postgresql server            | `5432`          |
+| `global.jobsdb.postgresqlUsername`           | Username of the Postgresql server        | `codacy`        |
+| `global.jobsdb.postgresqlDatabase`           | Database name of the Postgresql server   | `jobs`          |
+| `global.jobsdb.postgresqlPassword`           | Hostname of the Postgresql server        | None            |
+| `global.jobsdb.host`                         | Hostname of the Postgresql server        | None            |
+| `global.jobsdb.service.port`                 | Port of the Postgresql server            | `5432`          |
+| `activities.activitiesdb.postgresqlUsername` | Username of the Postgresql server        | `codacy`        |
+| `activities.activitiesdb.postgresqlDatabase` | Database name of the Postgresql server   | `jobs`          |
+| `activities.activitiesdb.postgresqlPassword` | Hostname of the Postgresql server        | None            |
+| `activities.activitiesdb.host`               | Hostname of the Postgresql server        | None            |
+| `activities.activitiesdb.service.port`       | Port of the Postgresql server            | `5432`          |
+| `hotspots-api.hotspotsdb.postgresqlUsername` | Username of the Postgresql server        | `codacy`        |
+| `hotspots-api.hotspotsdb.postgresqlDatabase` | Database name of the Postgresql server   | `hotspots`      |
+| `hotspots-api.hotspotsdb.postgresqlPassword` | Hostname of the Postgresql server        | None            |
+| `hotspots-api.hotspotsdb.host`               | Hostname of the Postgresql server        | None            |
+| `hotspots-api.hotspotsdb.service.port`       | Port of the Postgresql server            | `5432`          |
+| `githubEnterprise.hostname`                  | Hostname of githubEnterprise instance    | None            |
+| `githubEnterprise.protocol`                  | Protocol of githubEnterprise instance    | None            |
+| `githubEnterprise.port`                      | Port of githubEnterprise instance        | None            |
+| `githubEnterprise.isPrivateMode`             | PrivateMode of githubEnterprise instance | None            |
```

Comment:
Enable private mode when you have that setting enabled on your GithubEnterprise instance